orderlyweb_api/orderly_web_api.py

```diff
@@ -4,16 +4,14 @@
 
 
 class OrderlyWebAPI:
-    def __init__(self, base_url, token):
+    def __init__(self, base_url, auth_token):
```

Comment:
Sorry to be a pain, but I still don't find `auth_token` and `access_token` to be transparently different terms - can this one be `montagu_token`?

Reply:
It could theoretically be a Github token though, couldn't it?

Reply:
Would something like `auth_provider_token` be better?

Reply:
Ah of course, sorry I completely forgot about that! `auth_provider_token` would be perfect